### PR TITLE
[BACKLOG-23011] Prevent force stop after a safe stop has been started

### PR DESCRIPTION
**Pull request by mkambol**
https://jira.pentaho.com/browse/BACKLOG-23011

### CODE REVIEW

**Comment by mkambol** [comment]
@pentaho/ackbar

File: ui/src/main/java/org/pentaho/di/ui/spoon/trans/TransGraph.java [hunk 1]
```diff
@@ -4154,7 +4154,13 @@ public void run() {
   }
 
   public void stop() {
-    if ( ( running && !halting ) || safeStopping ) {
+    if ( safeStopping ) {
+      MessageBox messageBox = new MessageBox( shell, SWT.ICON_ERROR | SWT.OK );
```
Comment (kurtwalker) [inline ui/src/main/java/org/pentaho/di/ui/spoon/trans/TransGraph.java]: Should we use SimpleMessageDialog instead?

Comment (mkambol) [reply ui/src/main/java/org/pentaho/di/ui/spoon/trans/TransGraph.java]: Prolly

Comment (mkambol) [reply ui/src/main/java/org/pentaho/di/ui/spoon/trans/TransGraph.java]: The other message dialogs in TransGraph are all MessageBox, so I was going for consistency.  I'll send a screenshot to Stephanie to make sure it looks okay to her.

Comment (kurtwalker) [reply ui/src/main/java/org/pentaho/di/ui/spoon/trans/TransGraph.java]: ok.  :+1:

**Pull request merged**